Kickstarter-iOS/Views/CommentComposerView.swift

```diff
@@ -0,0 +1,129 @@
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Border {
+    static let margin: CGFloat = 15.0
+    static let height: CGFloat = 1.0
+  }
+
+  enum Avatar {
+    static let height: CGFloat = 44.0
+    static let margin: CGFloat = 18.0
+  }
+
+  enum InputContainer {
+    static let spacing: CGFloat = 12.0
+    static let margin: CGFloat = 18.0
+  }
+}
+
+final class CommentComposerView: UIView {
+  // MARK: - Properties
+
+  private let topBorderView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let inputContainerView: CommentInputContainerView = {
+    CommentInputContainerView()
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let onlyBackersLabel: UILabel = {
+    UILabel(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.isHidden .~ true
+  }()
+
+  private lazy var avatarImageView = {
```

Comment:
Same here, should we bind this to some view model logic on their actual avatar image?